### ISSUE TITLE
getter name conflict issue

### ISSUE THREAD

**Issue by FalconIA** [json-iterator/java#57]
```
public class JsonUtilsTest {

	@Test
	public void toJson() throws Exception {
		ResourceSupport obj= new ResourceSupport();
		JsonStream.serialize(obj);
	}

	public class ResourceSupport implements Identifiable<Long>, Serializable {

		@JsonIgnore
		@Override
		public Long getId() {
			return null;
		}
	}

	public interface Identifiable<ID extends Serializable> {
		ID getId();
	}
}
```

Get exception with message: `getter name conflict: id`

jsoniter version is 0.9.11

**Comment by taowen** [comment]
should be fixed now. will release a new version soon

**Comment by taowen** [comment]
fixed in 0.9.12
